test/PowerShellEditorServices.Test.Shared/Refactoring/Functions/ForeachFunction.ps1

```diff
@@ -0,0 +1,17 @@
+$x = 1..10
+
+function testing_files {
+    param (
+        $x
+    )
+    write-host "Printing $x"
+}
+
+foreach ($number in $x) {
+    testing_files $number
+
+    function testing_files {
+        write-host "------------------"
```

Comment:
Unhandled Internal Exception trying to rename write-host, needs to be handled either

Reply:
This is showing now due to the client side changes and not swallowing the error. It lead me to realize that CommandAst inputs are getting sent to the Variable Renamer not the function renamer which I have corrected since. It should show like the below now

![image](https://github.com/PowerShell/PowerShellEditorServices/assets/3089087/a6b80629-72b6-4d78-b9c1-a01951fd9056)